#SERE for RNA-Seq
#### *Introduction*

SERE [Single-parameter quality control and sample comparison for RNA-Seq] is a simple and reliable statistical method for the global assesment of pairs or large groups of RNA-Seq datasets. The method results in a single number which can be used to determine whether two or more libraries are faithful replicates or globally different. Further details of SERE can be found in [[1]](#ref_1).

Example
-----
Here we present an implementation of SERE in R. The code can be downloaded from <a id=" ">???link to source???</a>. As an example we run it on the supplementary data [] provided in [[2]](#ref_2).

References
-----
[1] <a id="ref_1" href="http://www.biomedcentral.com/1471-2164/13/524"> Schulze, S. K., Kanwar, R., Gölzenleuchter, M., Therneau, T. M., & Beutler, A. S. (2012). SERE: single-parameter quality control and sample comparison for RNA-Seq. *BMC genomics, 13*(1), 524. doi:10.1186/1471-2164-13-524 </a>

[2] <a id="ref_2" href="http://genome.cshlp.org/content/18/9/1509.long"> Marioni, J. C., Mason, C. E., Mane, S. M., Stephens, M., & Gilad, Y. (2008). RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays. *Genome research, 18*(9), 1509–17. doi:10.1101/gr.079558.108 </a>





